client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -165,6 +190,68 @@ export class Mentions extends React.Component {
 		editor.getBody().focus();
 	}
 
+	onKeyDown = ( e ) => {
+		const { keyCode } = e;
+
+		if ( includes( [ VK.DOWN, VK.UP ], keyCode ) ) {
+			const selectedIndex = this.getSelectedSuggestionIndex();
+			let nextIndex;
+
+			if ( selectedIndex !== null ) {
```

Comment:
this is another great place for an early-abort…

```js
if ( null === selectedIndex ) {
	return;
}
```

that will save us a level of indent which is always good